src/server.ts

```diff
@@ -71,14 +71,15 @@ export const start = (options: StartOptions) => {
   const schemaWithMiddleware = applyMiddleware(schema, logMiddleware, resolveRepoFromId, filterDirTraversal);
 
   const app = express();
+  const httpServer = http.createServer(app);
+
   const apolloServer = new ApolloServer({
     context: () => ({
       onAddRepoRequest: settings.onAddRepoRequest,
       onRemoveRepoRequest: settings.onRemoveRepoRequest,
       updateGitLoadingState: settings.updateGitLoadingState
     }),
     schema: schemaWithMiddleware,
-    subscriptions: false,
```

Comment:
Is this removed because we want to enable subscriptions or because `false` is the default value?

Reply:
> Is this removed because we want to enable subscriptions or because `false` is the default value?

@ron-rookout Apollo 3 no longer has the subscription package built in, so this flag no longer exists. Since we don't use subscriptions, this flag is not needed and there is no refactoring necessary in that regard